charts/karmada/templates/karmada-scheduler.yaml

```diff
@@ -50,6 +50,9 @@ spec:
             - --kubeconfig=/etc/kubeconfig
             - --bind-address=0.0.0.0
             - --secure-port=10351
+            - --feature-gates=Failover=true
```

Comment:
Failover has changed to Beta now and is enabled by default. So we do not need it anymore.

Reply:
the pr seem is not needed, so close it.